bitbots_body_behavior/src/bitbots_body_behavior/main.dsd

```diff
@@ -71,22 +71,22 @@ $TimerRunning + name:reorientation
 #GoalieBehavior
 $ClosestToBall
     YES --> $BallDangerous
-        NO --> @ChangeAction + action:positioning, @AvoidBallActive, @GoToBlockPosition, @LookAtFieldFeatures
+        NO --> @ChangeAction + action:positioning, @AvoidBallActive, @LookAtFieldFeatures, @GoToBlockPosition
         ELSE --> #GoAndKickBallMapGoal
-    NO --> @ChangeAction + action:positioning, @GoToBlockPosition, @LookAtFieldFeatures
+    NO --> @ChangeAction + action:positioning, @LookAtFieldFeatures, @GoToBlockPosition
 
 #StrikerRole
 $GoalieHandlingBall
-    YES --> @ChangeAction + action:positioning, @AvoidBallActive, @GoToPassAcceptPosition //don't go too far back
+    YES --> @ChangeAction + action:positioning, @AvoidBallActive, @LookAtFieldFeatures, @GoToPassAcceptPosition //don't go too far back
     NO --> #GoAndKickBallMapGoal
 
 #SupporterRole
 $PassStarted
-    YES --> @ChangeAction + action:positioning, @AvoidBallActive, @GoToPassAcceptPosition
-    NO --> @ChangeAction + action:positioning, @AvoidBallActive, @GoToPassPreparePosition
+    YES --> @LookAtBall, @ChangeAction + action:positioning, @AvoidBallInactive, @GoToPassAcceptPosition
```

Comment:
I don't think we want to do this here. Because, the pass may land on our side and we would walk right through it.

Reply:
Yes, but otherwise we will "run away" from the ball first before actually kicking it

Reply:
```suggestion
    YES --> @LookAtBall, @ChangeAction + action:positioning, @AvoidBallActive, @GoToPassAcceptPosition
```